index.html

```diff
@@ -1,113 +1,85 @@
+		<a href="https://github.com/terrestris/react-geo"><img alt="Fork me on GitHub" src="https://s3.amazonaws.com/github/ribbons/forkme_right_green_007200.png" style="position: absolute; top: 0; right: 0; border: 0; z-index:1000"></a>
+	</div><!-- Navbar -->
+	<nav class="navbar navbar-default">
+		<div class="container">
+			<div class="navbar-header">
+				<button class="navbar-toggle" data-target="#myNavbar" data-toggle="collapse" type="button"><span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span></button> <img src="assets/logo.svg" style="width: 70px;"> <a class="navbar-brand" href="">react-geo</a>
+			</div>
+			<div class="collapse navbar-collapse" id="myNavbar">
+				<ul class="nav navbar-nav navbar-center">
+					<li>
+						<a href="./docs/@terrestris/react-geo/5.4.0/index.html">Docs</a>
+					</li>
+					<li>
+						<a href="./examples/index.html">Examples</a>
+					</li>
+					<li id="gh-link">
+						<a href="https://github.com/terrestris/react-geo">Github</a>
+					</li>
+				</ul>
+			</div>
+		</div>
+	</nav><!-- First Container -->
+	<div class="container-fluid text-center">
+		<div class="carousel slide" id="myCarousel">
+			<!-- Wrapper for slides -->
+			<div class="carousel-inner" role="listbox">
+				<div class="text-center item active">
+					<img class="img-responsive img-circle margin" src="assets/logo.svg" style="display:inline" width="200">
+					<h3>A set of geo related modules.<br>
+					To use in combination with <a href="https://reactjs.org/">react</a>, <a href="https://ant.design/">antd</a> and <a href="https://openlayers.org">ol</a>.</h3>
+					<div class="carousel-caption"></div>
+				</div>
+				<div class="item">
+					<div class="item2">
+						<h3>Create your own WebGIS easily with the power of reactjs, andt and Openlayers</h3>
+					</div>
+					<div class="carousel-caption"></div>
+				</div>
+				<div class="item">
+					<h3>Check out <a href="examples/index.html">examples</a> and <a href="./wiki/Feature-List">feature list</a> for full functionality of react-geo.</h3><br>
+					<img class="rounded img-exp" src="assets/geoutil.png" style="vertical-align:middle" width="286px"> <span style="font-size: 22px;">GeometryUtil features<br>
+					<span style="font-size: 22px;">Map components</span><img class="rounded img-exp" src="assets/distance.png"></span>
+					<div class="carousel-caption"></div>
+				</div><!-- Left and right controls -->
+				 <a class="left carousel-control" data-slide="prev" href="#myCarousel" role="button"><span aria-hidden="true" class="glyphicon glyphicon-chevron-left"></span> <span class="sr-only">Previous</span></a> <a class="right carousel-control" data-slide="next" href="#myCarousel" role="button"><span aria-hidden="true" class="glyphicon glyphicon-chevron-right"></span> <span class="sr-only">Next</span></a>
+			</div>
+		</div><!-- Footer -->
+		<footer class="container-fluid-footer bg-4 text-center"></footer>
+	</div>
```

Comment:
Note: The `index.html` will be generated, so all changes in here will be overwritten!

Reply:
Okay, I tidied up the css, should be fine now.